Add initial version of 3scale alert SOPs included on 3scale v2.9 (#9)
3scale/3scale-Operations: sops/alerts/backend_listener_5xx_requests_high.adoc

:toc:
:toc-placement!:

= ThreescaleBackendListener5XXRequestsHigh

toc::[]

== Description

* This alert will trigger if the number of `backend-listener` HTTP 5XX requests is greater than a given threshold

== Troubleshooting

* Check if `backend-listener` pods might be failing
* Check if `backend-listener` pods might not be able to deal with incoming traffic due to high CPU usage
- You may need to consider scaling horinzontally `backend-listener` deployment
- You may need to consider increasing `backend-listener` deployment resources requests/limits

> * Check if backend's redis is having issues

Author: ```suggestion
* Check if backend's redis is having issues
* Check if the current traffic is normal based on the historical data. An unexpected change in the traffic pattern can be legit, and will require scaling up, but it also can be due to an abnormal or malicious traffic

```

:toc:
:toc-placement!:

= ThreescaleBackendListener5XXRequestsHigh

toc::[]

== Description

* This alert will trigger if the number of `backend-listener` HTTP 5XX requests is greater than a given threshold

== Troubleshooting

* Check if `backend-listener` pods might be failing

> * Check if `backend-listener` pods might not be able to deal with incoming traffic (too much cpu usage causing slowliness?)

Reviewer: Same rewording as the latency example. 
```suggestion
* Check if `backend-listener` pods might not be able to deal with incoming traffic due to high CPU usage
```

- You may need to consider scaling horinzontally `backend-listener` deployment
- You may need to consider increasing `backend-listener` deployment resources requests/limits

> * Check if redis is having some issues

Author: ```suggestion
* Check if backend's redis is having some issues
```

> * Check if backend's redis is having some issues

Author: ```suggestion
* Check if backend's redis is having issues
```

> 

Author: ```suggestion
```

> 

Reviewer: Same as the latency, the increase can be due to an abnormal or malicious traffic, and in this case, also just invalid requests.

```suggestion
* Check if the current traffic is normal based on the historical data. An unexpected increase of traffic can be legit, and will require scaling up, but it also be part of an attack, and will require countermeasures to block the malicious requests .

```
